gsoc/suborg_guidelines.md

```diff
@@ -13,20 +13,22 @@ slots we will get till the student selection process ends. However,
 it's our intention to keep the process fair for the suborganisations
 and students.
 
-## I want to have to become a sub-org admin, what do I do?
+## I want to become a sub-org admin, what do I do?
 
-First you need to be a member of the OpenAstronomy team. That process
+First, we recommend you to carefully read [Google's notes for first year organizations][Google-notes],
+which provides a general idea about the goals and concepts underpinning the Google Summer of Code program.
+Then, you need to be a member of the OpenAstronomy team. That process
 is simple (and free!). You need to be an open-development organisation
 and be related with astronomy. Then make a pull-request to [our repository][OA repository]
-adding a logo, a short description, etc. 
+adding a logo, a short description, etc.
 Follow the instructions in the example at the bottom of [the `memembers.yaml` file][members PR].
```

Comment:
I feel the OA council should come up with a better set of rules of what needed to be come a member. But don't have perfect be the enemy of good, let's go ahead with this for now as it's better already than it was.